src/core/ServiceAccount.ts

```diff
@@ -40,6 +40,11 @@ export class ServiceAccount {
    */
   public config?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
 
+  /**
+   * Widget configurations for the serviceAccount
+   */
+  public widgetConfig?: WidgetConfig;
+
```

Comment:
I just have a question. I wonder why `widgetConfig` property doesn't belong to `config` property 🤔

Reply:
the config ones are the critical and secured data that are encrypted. I wanted to make a difference between the both since the widget configurations won't be encrypted

Reply:
Ok 👍